docs/serving/debugging-application-issues.md

```diff
@@ -43,46 +43,48 @@ kubectl get route <route-name> --output yaml
 
 The `conditions` in `status` provide the reason if there is any failure. For
 details, see Knative
-[Error Conditions and Reporting](https://github.com/knative/serving/blob/master/docs/spec/errors.md)(currently
+[Error Conditions and Reporting](https://github.com/knative/docs/blob/master/docs/serving/spec/knative-api-specification-1.0.md#error-signalling)(currently
 some of them are not implemented yet).
 
-### Check ClusterIngress/Istio routing
+### Check Ingress/Istio routing
 
-Run the following command to list all the cluster ingress, with their labels
+Run the following command to list all the ingress, with their labels:
 
 ```shell
-kubectl get clusteringress -o=custom-columns='NAME:.metadata.name,LABELS:.metadata.labels'
-NAME                   LABELS
-helloworld-go-h5kd4    map[serving.knative.dev/route:helloworld-go serving.knative.dev/routeNamespace:default]
+kubectl get ingresses.networking.internal.knative.dev -o=custom-columns='NAME:.metadata.name,LABELS:.metadata.labels'
+NAME            LABELS
+helloworld-go   map[serving.knative.dev/route:helloworld-go serving.knative.dev/routeNamespace:default serving.knative.dev/service:helloworld-go]
 ```
 
 The labels `serving.knative.dev/route` and `serving.knative.dev/routeNamespace`
-will tell exactly which Route a ClusterIngress is a child resource of. Find the
-one corresponding to your Route. If a ClusterIngress does not exist, the route
+will tell exactly which Route an Ingress is a child resource of. Find the
+one corresponding to your Route. If an Ingress does not exist, the route
 controller believes that the Revisions targeted by your Route/Service isn't
 ready. Please proceed to later sections to diagnose Revision readiness status.
 
 Otherwise, run the following command to look at the ClusterIngress created for
 your Route
 
 ```
-kubectl get clusteringress <CLUSTERINGRESS_NAME> --output yaml
+kubectl get ingresses.networking.internal.knative.dev <INGRESS_NAME> --output yaml
 ```
 
-particularly, look at the `status:` section. If the ClusterIngress is working
+particularly, look at the `status:` section. If the Ingress is working
 correctly, we should see the condition with `type=Ready` to have `status=True`.
 Otherwise, there will be error messages.
 
-Now, if ClusterIngress shows status Ready, there must be a corresponding
+You can replace resource `ingresses.networking.internal.knative.dev` with `ClusterIngress` if you are using Knative release older than 0.9.
```

Comment:
We do not need to reference past versions. Our content is published by release and therefore, users of past versions should view the previous/corresponding doc set release.
```suggestion
```